2016/proposals/chris-medrela.md

```diff
@@ -0,0 +1,461 @@
+![discussion-session-detailed-1](https://cloud.githubusercontent.com/assets/1645996/13723295/3d0e7582-e860-11e5-94ae-7e38a02c081a.png)
+
+- This view lets the Discussion Leader customize emails:
+![discussion-session-detailed-2](https://cloud.githubusercontent.com/assets/1645996/13723297/454654d6-e860-11e5-8137-93c57d4a9325.png)
+
+- At this moment, trainees can register for Checkout Session.
+
+- After the Checkout Session, the Examiner indicates in Amy who has passed and who has failed (the interface would be similar to the detailed view of Discussion Session). Amy mails the Trainees with instructions on what to do next.
+
+    - Note that Admins and Head of Training are not engaged at all.
+
+- Amy generates PDF certificates. Those certificates are hosted as static files on Amy (much easier than integrating Amy with github).
+
+- New instructors send their biography as well as photo in Amy; and are asked to join `discuss` as well as `instructors` list. After sending it, they can download their certificate from Amy.
+
+## Milestone 2: Managing DC Checkout Process
+
+This Milestone is an easy extension of the previous one. Therefore, I believe there is no need for early users.
+
+At the end of previous Milestone, trainees can view progress of their SWC Checkout Process. In this Milestone, we'll introduce a second page with progress of DC Checkout Process, that will be very similar to the first one.
+
+## Milestone 3: Managing Recruiting Candidates
+
+That is managing everything before and during an Instructor Training -- Recruiting Candidates, Matching them to Instructor Trainings and running those trainings.
+
+All Candidates will be represented in `Person` table, from the beginning of the workflow. For simplicity, trainees can login using their emails, so they don't have to choose username.
+
+The new workflow is:
+
+- A Candidate or a Lead Candidate of a group of Candidates uses Registration Form to create his account. This form contains only one field: email address. After submitting the form, Amy sends an email with link to another form, where the Candidate can choose password and enter some basic information about him (something like `update_profile`). The Registration Form accepts emails addresses that are already present in Amy Database. In that case, it serves as password recovery.
+
+- The Candidate logs into Amy and uses Application Form to apply for an Instructor Training. This form is used for both individual Candidates as well as groups of Candidates. In the case of group applications, the Lead Candidate needs to enter names and email addresses of other Candidates. After submitting the form, a new account is created for each Candidate, but no password is set yet. For simplicity, UI allows the Candidate to request only one Instructor Training.
```

Comment:
They don't confirm at this moment. I think it's a bit of hassle to require every Candidate to register and confirm. In the process of negotiating Instructor Training date, all Candidates will be manually mailed to confirm that they will attend.

I'm not sure if this is actually good idea since it requires manual work, but I wonder if automating sending those emails will make it easier for Admins. What do you think about that?